[ 2024-12-20 07:10:15,818 ] 20 root - INFO - Initiate the data ingestion
[ 2024-12-20 07:10:52,790 ] 65 root - INFO - Performed train test split on the dataframe
[ 2024-12-20 07:10:52,790 ] 67 root - INFO - Exited split_data_as_train_test method of Data_Ingestion class
[ 2024-12-20 07:10:52,790 ] 75 root - INFO - Exporting train and test file path.
[ 2024-12-20 07:10:52,871 ] 84 root - INFO - Exported train and test file path.
[ 2024-12-20 07:10:52,871 ] 22 root - INFO - Data Initiation Completed
[ 2024-12-20 07:10:52,875 ] 26 root - INFO - Initiate the data Validation
[ 2024-12-20 07:10:52,927 ] 32 root - INFO - Required number of columns:2
[ 2024-12-20 07:10:52,928 ] 33 root - INFO - Data frame has columns:31
[ 2024-12-20 07:10:52,928 ] 32 root - INFO - Required number of columns:2
[ 2024-12-20 07:10:52,928 ] 33 root - INFO - Data frame has columns:31
[ 2024-12-20 07:10:53,085 ] 28 root - INFO - data Validation Completed
[ 2024-12-20 07:10:53,086 ] 31 root - INFO - data Transformation started
[ 2024-12-20 07:10:53,086 ] 63 root - INFO - Entered initiate_data_transformation method of DataTransformation class
[ 2024-12-20 07:10:53,086 ] 65 root - INFO - Starting data transformation
[ 2024-12-20 07:10:53,115 ] 48 root - INFO - Entered get_data_trnasformer_object method of Trnasformation class
[ 2024-12-20 07:10:53,115 ] 53 root - INFO - Initialise KNNImputer with {'missing_values': nan, 'n_neighbors': 3, 'weights': 'uniform'}
[ 2024-12-20 07:10:53,128 ] 46 root - INFO - Entered the save_object method of MainUtils class
[ 2024-12-20 07:10:53,132 ] 50 root - INFO - Exited the save_object method of MainUtils class
[ 2024-12-20 07:10:53,132 ] 46 root - INFO - Entered the save_object method of MainUtils class
[ 2024-12-20 07:10:53,135 ] 50 root - INFO - Exited the save_object method of MainUtils class
[ 2024-12-20 07:10:53,137 ] 35 root - INFO - data Transformation completed
[ 2024-12-20 07:10:53,137 ] 37 root - INFO - Model Training sstared
[ 2024-12-20 07:16:17,105 ] 870 urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=5, read=5, redirect=5, status=5)) after connection broken by 'SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:2406)')': /krishnaik06/networksecurity.mlflow/api/2.0/mlflow-artifacts/artifacts/39dd78e691c14027a66f1293747022d5/c5b19993135c413585168ccc37179ac6/artifacts/model/model.pkl
[ 2024-12-20 07:26:02,056 ] 870 urllib3.connectionpool - WARNING - Retrying (Retry(total=4, connect=5, read=5, redirect=5, status=5)) after connection broken by 'SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:2406)')': /krishnaik06/networksecurity.mlflow/api/2.0/mlflow-artifacts/artifacts/39dd78e691c14027a66f1293747022d5/c5b19993135c413585168ccc37179ac6/artifacts/model/model.pkl
